### PR TITLE
Insert from file

### PR DESCRIPTION
**Pull request by Maksim-Burtsev**

### CODE REVIEW

**Comment by maximdanilchenko** [comment]
Btw thanks for the contribution! Very useful method.

**Comment by Maksim-Burtsev** [comment]
@maximdanilchenko should i change something so that you merge this PR?

**Comment by maximdanilchenko** [comment]
Sorry, made a comment, but forgot to submit review

File: aiochclient/client.py [hunk 1]
```diff
@@ -415,6 +415,53 @@ async def cursor(self, query: str, *args) -> AsyncGenerator[Record, None]:
         async for row in self.iterate(query, *args):
             yield row
 
+    async def insert_file(
+        self,
+        query: str,
+        filepath: str,
```
[the remark this answers is not in the document]
Comment (Maksim-Burtsev) [reply aiochclient/client.py]: Great idea!

[the remark this answers is not in the document]
Comment (Maksim-Burtsev) [reply aiochclient/client.py]: I made updates and now method has `file_obj` param

**Pull request merged**